nbbuild/antsrc/org/netbeans/nbbuild/GetModuleName.java

```diff
@@ -28,20 +28,28 @@
  * @author Michal Zlamal
  */
 public class GetModuleName extends Task {
-    String name = null;
-    File root = null;
+    String name;
+    String id;
+    File root;
 
     // XXX this is a lousy attr name; conventional for such attrs to
     // end in 'property' so you realize they refer to a property name
     public void setName (String name) {
         this.name = name;
     }
 
-    /** Root directory of the whole project - ${nb_all} */
-    public void setRoot( File root ) {
+    public void setId (String id) {
```

Comment:
This is introducing a module `id` - e.g. not full path, but just the shortened name as used in `cluster.properties` & co.